recipes/LibriSpeech/ASR/CTC/extra_requirements.txt

```diff
@@ -1 +1,4 @@
+# k2 # It is better to install k2 with the procedure listed here: https://k2-fsa.github.io/k2/installation/from_wheels.html
+kaldilm==1.15
```

Comment:
Hmmm what is `kaldilm` ?

Reply:
https://github.com/csukuangfj/kaldilm KaldiLM is a tool to create the ARPA to FST mapping that k2 needs to read the arpa file.
It is a C++ to python wrapper, without any kaldi installation dependency.